docs/src/modules/components/AdGuest.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import Portal from '@mui/material/Portal';
 import { AdContext } from 'docs/src/modules/components/AdManager';
 
-export default function AdGuest(props) {
+export default function AdGuest({ classSelector = '.description', ...props }) {
```

Comment:
The `classSelector` prop allows other elements to be connected to the ad, for example the tabs element, instead of the description. By default (without the prop being provided), the ad will be placed below the description - same as before.